From eb507f5505af4c88537b6ede1413de02ba9d7eb5 Mon Sep 17 00:00:00 2001
From: dave14305 <44532942+dave14305@users.noreply.github.com>
Date: Tue, 7 Jul 2026 11:55:23 -0400
Subject: [PATCH] webui: restart mastiff when toggling Asusnat tunnel (#892)

Disabling or enabling the Asusnat tunnel should trigger a restart of the affected services.

(cherry picked from commit 5a6efff1fcdb265f05da16fa2bdb7f051635e5c2)
---
 release/src/router/www/Tools_OtherSettings.asp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src/router/www/Tools_OtherSettings.asp b/release/src/router/www/Tools_OtherSettings.asp
index 0dfbb292cd..65d7718cf4 100644
--- a/release/src/router/www/Tools_OtherSettings.asp
+++ b/release/src/router/www/Tools_OtherSettings.asp
@@ -91,7 +91,7 @@ function applyRule(){
 <input type="hidden" name="next_page" value="Tools_OtherSettings.asp">
 <input type="hidden" name="modified" value="0">
 <input type="hidden" name="action_mode" value="apply">
-<input type="hidden" name="action_script" value="restart_conntrack">
+<input type="hidden" name="action_script" value="restart_conntrack;restart_mastiff">
 <input type="hidden" name="action_wait" value="5">
 <input type="hidden" name="first_time" value="">
 <input type="hidden" name="SystemCmd" value="">
-- 
2.25.1

